gateway-eth-ts/src/service/GatewayTsForwarder.ts

```diff
@@ -129,11 +130,12 @@ export class GatewayTsForwarder extends GatewayTsInternal<
   async issue(
     owner: string,
     network: bigint,
-    expiry?: BigNumberish,
-    bitmask?: BigNumberish,
-    charge?: Charge
+    expiry: BigNumberish = 0,
```

Comment:
Is 0 an okay default? Wouldn't that mean the pass expires immediately

Reply:
Yep this is fine. If the expire time is zero that means the pass does not expire.